src/content/en/2024/sustainability.md

```diff
@@ -510,7 +510,7 @@ Let's look at how the usage of these formats has changed from 2022 to 2024:
   description="A bar chart showing the comparison of image formats adoption between 2022 and 2024. It shows that the jpg adoption is 20% less in 2024, while png increased 1%, gif is up by 6%, webp is 34% more, svg also increased with 36% more, ico is 17% less and avif is 386% more.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQCSP87WE9bhFjIICxcrpIEGQlz3xBq33-ODZ8e91XSLUbLvAZjk25GhOdDtFIZCzPcS-VrSygr7pmz/pubchart?oid=2128858223&format=interactive",
   sheets_gid="1939630368",
-  sql_file="TODO get from Media chapter"
+  sql_file=""
```

Comment:
keeping it empty because it looks like there is no query for this. Someone from the Media chapter probably manually comparing the result of media/media_formats.sql between 2024 and 2022